### ISSUE TITLE
Not working if iTerm2 sends text at start

### ISSUE THREAD

**Issue by olalonde** [TheNewNormal/kube-solo-osx#147]
<img width="649" alt="screenshot 2016-10-28 16 08 36" src="https://cloud.githubusercontent.com/assets/416585/19824959/d5502098-9d28-11e6-9735-7dc1e4145238.png">

![mcfsk7uiwa](https://cloud.githubusercontent.com/assets/416585/19824967/e4830bde-9d28-11e6-8553-dc0c7f9f8640.gif)

Is there any way to get Kube-Solo to use the window started by tmux and/or be able to configure a non default iTerm2 profile which will be used?

**Comment by rimusz** [comment]
The default `Terminal` would be easier to be used instead of `tmux`
The required terminal app could be stored in some settings file and used opening shell scripts

**Comment by olalonde** [comment]
Indeed. Trying to make it work with tmux seems like a bit of a PITA. Another option would be to see if a flag can be passed to `open -a iTerm.app "$1"/first-init.command` to tell iTerm to ignore the "send text at start" setting. That would probably be ideal. What do you think? (asked here https://gitlab.com/gnachman/iterm2/issues/5288)

**Comment by olalonde** [comment]
Looks like another way would be to chose a profile called "kube-solo" if it exists and otherwise use the default profile: https://www.iterm2.com/documentation-scripting.html. That way, I could create a "kube-solo" profile which doesn't send text at start.

**Comment by olalonde** [comment]
Oh, looks like there will be a new option in iTerm to "disabled text at start when opening a script" https://gitlab.com/gnachman/iterm2/issues/5288. 👍

**Comment by rimusz** [comment]
ok, cool
